From 2685716409c560cdf94aee6262ed8c32ad137133 Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Fri, 22 Nov 2019 09:24:58 +0800
Subject: [PATCH] setup.py: remove maximum version requirements

Remove maximum version requirements so that docker-compose will not
require old version recipes.

Upstream-Status: Inappropriate [OE Specific]

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 setup.py | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/setup.py b/setup.py
index 422ba54..c2b4556 100644
--- a/setup.py
+++ b/setup.py
@@ -30,16 +30,16 @@ def find_version(*file_paths):
 
 
 install_requires = [
-    'cached-property >= 1.2.0, < 2',
-    'docopt >= 0.6.1, < 0.7',
-    'PyYAML >= 3.10, < 4',
-    'requests >= 2.6.1, != 2.11.0, != 2.12.2, != 2.18.0, < 2.19',
-    'texttable >= 0.9.0, < 0.10',
-    'websocket-client >= 0.32.0, < 1.0',
-    'docker >= 3.3.0, < 4.0',
-    'dockerpty >= 0.4.1, < 0.5',
-    'six >= 1.3.0, < 2',
-    'jsonschema >= 2.5.1, < 3',
+    'cached-property >= 1.2.0',
+    'docopt >= 0.6.1',
+    'PyYAML >= 3.10',
+    'requests >= 2.6.1, != 2.11.0, != 2.12.2, != 2.18.0',
+    'texttable >= 0.9.0',
+    'websocket-client >= 0.32.0',
+    'docker >= 3.3.0',
+    'dockerpty >= 0.4.1',
+    'six >= 1.3.0',
+    'jsonschema >= 2.5.1',
 ]
 
 
@@ -56,7 +56,7 @@ extras_require = {
     ':python_version < "3.5"': ['backports.ssl_match_hostname >= 3.5'],
     ':python_version < "3.3"': ['ipaddress >= 1.0.16'],
     ':sys_platform == "win32"': ['colorama >= 0.3.9, < 0.4'],
-    'socks': ['PySocks >= 1.5.6, != 1.5.7, < 2'],
+    'socks': ['PySocks >= 1.5.6, != 1.5.7'],
 }
 
 
-- 
2.17.1

